infretis/tools/rec_error.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+
+def rec_blocks(r):
+    b = []
+    for i in range(len(r)):
+        if i == 0:
+            b.append(r[i])
+        else:
+            b.append((i + 1) * r[i] - i * r[i - 1])
+    return b
+
+
+def rec_block_errors(runav, minblocks):
+    maxbll = int(len(runav) / minblocks)  # maximum block length
+    bestav = runav[-1]  # most accurate average we have
+    rel_errors = []
+    for bloclengtgh in range(1, maxbll + 1):
+        n = bloclengtgh
+        runav_red = runav[n - 1 :: n]  # take every nth value of the array
+        blocks = rec_blocks(runav_red)
+        sum_qudiff = sum(
```

Comment:
This can probably be something like `sum_qudiff = np.sum((blocks - bestav) ** 2)` if the rest used numpy.